migrations/3_monetary_policy.js

```diff
@@ -8,7 +8,8 @@ const Share = artifacts.require('Share');
 const IERC20 = artifacts.require('IERC20');
 const MockDai = artifacts.require('MockDai');
 
-const Oracle = artifacts.require('Oracle')
+const BondOracle = artifacts.require('Oracle')
```

Comment:
Basically, @truffle/artifact shares one definition file. 
So if you trying to deploy with same artifact, It can be overwritten by each other.

Reply:
I'll suggest you to save artifact manually
Please check `1_initial_migration.js`.

Reply:
```
  const BondOracle = await deployer.deploy(
    Oracle,
    uniswap.address,
    cash.address,
    dai.address,
    HOUR,
    ORACLE_START_DATE
  );
  const SeigniorageOracle = await deployer.deploy(
    Oracle,
    uniswap.address,
    cash.address,
    dai.address,
    DAY,
    ORACLE_START_DATE
  );
  ```

Reply:
@fr-jerry-smith Does this works then?

Reply:
Oh it works?

Reply:
why not?

Reply:
LOL let's get it good works